src/java/org/apache/cassandra/db/compaction/unified/Controller.java

```diff
@@ -598,10 +598,13 @@ public static Map<String, String> validateOptions(Map<String, String> options) t
             try
             {
                 long sizeInBytes = FBUtilities.parseHumanReadableBytes(s);
+                if (sizeInBytes < 0)
+                    throw new ConfigurationException(String.format("Invalid configuration, %s should be greater than 0 (zero)",
+                                                                   MIN_SSTABLE_SIZE_OPTION));
                 // zero is a valid option to disable featue
                 int limit = (int) Math.ceil(targetSSTableSize * INVERSE_SQRT_2);
-                if (sizeInBytes != 0 && sizeInBytes < limit)
-                    throw new ConfigurationException(String.format("Invalid configuration, %s should be 0 (zero) or greater than or equal to: %sB",
+                if (sizeInBytes != 0 && sizeInBytes >= limit )
```

Comment:
nit: `limit` will always be > 0 since `MIN_TARGET_SSTABLE_SIZE` is 1MiB, so checking that `sizeInBytes != 0` here is redundant.

Reply:
@Claudenw  I will start the builds once this is addressed. Thank you.